ibm_catalog.json

```diff
@@ -274,6 +278,9 @@
                 {
                   "key": "auto_scaling"
                 },
+                {
+                  "key": "backup_crn"
+                },
```

Comment:
what about the other vars you added? `existing_backup_kms_instance_crn` and `existing_backup_kms_key_crn`